pkg/util/testing/wrappers.go

```diff
@@ -238,6 +243,19 @@ func (p *PodSetWrapper) Request(r corev1.ResourceName, q string) *PodSetWrapper
 	return p
 }
 
+func (p *PodSetWrapper) Limit(r corev1.ResourceName, q string) *PodSetWrapper {
+	if p.Template.Spec.Containers[0].Resources.Limits == nil {
+		p.Template.Spec.Containers[0].Resources.Limits = corev1.ResourceList{}
+	}
+	p.Template.Spec.Containers[0].Resources.Limits[r] = resource.MustParse(q)
+	return p
+}
+
+func (p *PodSetWrapper) Image(image string) *PodSetWrapper {
```

Comment:
Why need this?

Reply:
To resolve conflicts with main branch,since now I merged this part into my PR,there're no conflicts any more.